CMakeLists.txt

```diff
@@ -84,7 +84,7 @@ if(${CMAKE_VERSION} VERSION_GREATER_EQUAL 3.17)
     cmake_policy(SET CMP0099 OLD)
 endif()
 
-find_program(PYTHON_EXECUTABLE NAMES python3)
+find_program(PYTHON_EXECUTABLE NAMES python3 python)
```

Comment:
We should set this variable for all of cmake, not only vpnglean. Fine to do it as a follow up though.

Reply:
@brizental IIRC, that's exactly what we do - this is the top level `CMakeLists.txt` and it gets set for the entire project.

Reply:
LOL, duh! I was confused because the title of the PR says vpnglean.... Sorry!